gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
I use many, currently vs code. What I meant is that I don't use the integrated debugger -- typically when I need a debugger I just do the normal node flags and open up chrome.

Nice to see you found what you need and your preferred debugging solution is still available to you!